src/platform_impl/linux/wayland/window/mod.rs

```diff
@@ -363,6 +363,11 @@ impl Window {
         self.maximized.load(Ordering::Relaxed)
     }
 
+    #[inline]
+    pub fn is_visible(&self) -> bool {
+        true
```

Comment:
I'm stronly agains this API in it's current form. If anything it should return `Option<bool>`, since on Wayland you don't known that. And you also shouldn't rely on it to draw in your code, since it's just bad.

Reply:
I don't mind `Option<bool>` but this was just meant as a placeholder because I can only provide implementation of Windows and macOS.